taskcat/_template_params.py

```diff
@@ -65,6 +65,25 @@ def transform_parameter(self):
         # - $[taskcat_gets3contents]
         # - $[taskcat_geturl]
         for param_name, param_value in self._param_dict.items():
+            if isinstance(param_value, list):
+                _results_list = []
+                _nested_param_dict = {}
+                for idx, value in enumerate(param_value):
+                    _nested_param_dict[idx] = value
+                nested_pg = ParamGen(
+                    _nested_param_dict,
+                    self.bucket_name,
+                    self.region,
+                    self._boto_client,
+                    self.az_excludes,
+                )
+                nested_pg.transform_parameter()
+                for result_value in nested_pg.results.values():
+                    _results_list.append(result_value)
+                self.param_value = _results_list
+                self.results.update({param_name: _results_list})
+                return
```

Comment:
won't this return stop additional parameters from being processed ?

Reply:
Good catch. That should be a 'continue'.